### PR TITLE
planner: update SPM doc

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by winoros — APPROVED** [review]

File: sql-plan-management.md [hunk 1]
```diff
@@ -476,6 +476,54 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 利用 `Statement Summary` 表获取需要绑定的查询
+
+[`Statement Summary`](/statement-summary-tables.md) 的表中存放了近期的 SQL 相关的执行信息，如延迟、执行次数、对应计划等，通过编写特定的查询访问这些表，可以快速找到需要绑定的查询。
+
+你可以通过查询 `Statement Summary` 表得到符合条件查询的 `plan_digest`，再使用[根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定)快速创建绑定。
+
+下面是一个实例，此实例会查找过去 2 周执行次数超过 10 次，执行计划不稳定且未被绑定的 Select 语句，按照执行次数排序，将执行次数前 100 的查询绑定到对应的查询延迟最低的计划上：
```
Comment (hfxsd) [inline sql-plan-management.md:485]:
```suggestion
[`Statement Summary`](/statement-summary-tables.md) 的表中存放了近期的 SQL 相关的执行信息，如延迟、执行次数、对应计划等。你可以通过查询 `Statement Summary` 表得到符合条件查询的 `plan_digest`，然后[根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定)。

以下示例查找过去 2 周执行次数超过 10 次、执行计划不稳定且未被绑定的 SELECT 语句，按照执行次数排序，将执行次数前 100 的查询绑定到对应的查询延迟最低的计划上。
```

File: sql-plan-management.md [hunk 2]
```diff
@@ -476,6 +476,54 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 利用 `Statement Summary` 表获取需要绑定的查询
```
Comment (Oreoxmt) [inline sql-plan-management.md:479]:
```suggestion
## 利用 Statement Summary 表获取需要绑定的查询
```

File: sql-plan-management.md [hunk 3]
```diff
@@ -476,6 +476,54 @@ SHOW binding_cache status;
...
+
+[`Statement Summary`](/statement-summary-tables.md) 的表中存放了近期的 SQL 相关的执行信息，如延迟、执行次数、对应计划等，通过编写特定的查询访问这些表，可以快速找到需要绑定的查询。
```
Comment (Oreoxmt) [inline sql-plan-management.md:481]:
```suggestion
[Statement Summary](/statement-summary-tables.md) 表记录了近期 SQL 相关的执行信息，如延迟、执行次数和相应执行计划等。通过针对这些表执行特定查询，可以快速找到需要绑定的查询。
```

File: sql-plan-management.md [hunk 4]
```diff
@@ -476,6 +476,54 @@ SHOW binding_cache status;
...
+
+你可以通过查询 `Statement Summary` 表得到符合条件查询的 `plan_digest`，再使用[根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定)快速创建绑定。
```
Comment (Oreoxmt) [inline sql-plan-management.md:483]:
```suggestion
你可以通过查询 Statement Summary 表获取满足条件的查询的 `plan_digest`，再使用[根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定)快速创建绑定。
```

File: sql-plan-management.md [hunk 5]
```diff
@@ -476,6 +476,54 @@ SHOW binding_cache status;
...
+WITH stmts AS (                                                -- Gets all information
+  SELECT * FROM INFORMATION_SCHEMA.CLUSTER_STATEMENTS_SUMMARY
+  UNION ALL
+  SELECT * FROM INFORMATION_SCHEMA.CLUSTER_STATEMENTS_SUMMARY_HISTORY 
+),
+best_plans AS (
+  SELECT plan_digest, `digest`, avg_latency, 
+  CONCAT('create global binding from history using plan digest "', plan_digest, '"') as binding_stmt 
+  FROM stmts t1
+  WHERE avg_latency = (SELECT min(avg_latency) FROM stmts t2   -- The plan with the loweset query latency
```
Comment (Oreoxmt) [inline sql-plan-management.md:497]:
```suggestion
  WHERE avg_latency = (SELECT min(avg_latency) FROM stmts t2   -- The plan with the lowest query latency
```

File: sql-plan-management.md [hunk 6]
```diff
@@ -476,6 +476,54 @@ SHOW binding_cache status;
...
+FROM stmts, best_plans
+WHERE stmts.`digest` = best_plans.`digest`
+  AND summary_begin_time > DATE_SUB(NOW(), interval 14 day)    -- Executed in the past 2 weeks
+  AND stmt_type = 'Select'                                     -- Only consider select statements
+  AND schema_name NOT IN ('INFORMATION_SCHEMA', 'mysql')       -- Not an internal query
+  AND plan_in_binding = 0                                      -- No binding yet
+GROUP BY stmts.`digest`
+  HAVING COUNT(DISTINCT(stmts.plan_digest)) > 1                -- This query is unstable. It has more than 1 plan.
+         AND SUM(exec_count) > 10                              -- High-frequency, and has been executed more than 10 times.
+ORDER BY SUM(exec_count) DESC LIMIT 100;                       -- Top 100 high-grequency queries.
```
Comment (Oreoxmt) [inline sql-plan-management.md:513]:
```suggestion
ORDER BY SUM(exec_count) DESC LIMIT 100;                       -- Top 100 high-frequency queries.
```

File: sql-plan-management.md [hunk 7]
```diff
@@ -227,7 +227,7 @@ CREATE BINDING FOR SELECT * FROM t WHERE a > 1 USING SELECT * FROM t use index(i
 以下为根据历史执行计划创建绑定的注意事项：
 
 - 该功能是根据历史的执行计划生成 hint 而实现的绑定，历史的执行计划来源是 [Statement Summary Tables](/statement-summary-tables.md)，因此在使用此功能之前需开启系统变量 [`tidb_enable_stmt_summary`](/system-variables.md#tidb_enable_stmt_summary-从-v304-版本开始引入)。
-- 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，目前还不支持通过历史执行计划进行绑定。
+- 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，自动生成的 `Hint` 不够完备，可能导致无法完全固定住计划，对于这类情况在创建时会产生 warning。
```
Comment (Oreoxmt) [inline sql-plan-management.md:230]:
```suggestion
- 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，自动生成的 hint 不够完备，可能导致无法完全固定住计划，对于这类情况在创建时会产生 warning。
```

File: system-variables.md [hunk 8]
```diff
@@ -2910,6 +2910,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 这个变量用来设置基于负载的 replica read 的触发阈值。当 leader 节点的预估排队时间超过阈值时，TiDB 会优先从 follower 节点读取数据。格式为时间，例如 `"100ms"` 或 `"1s"`。详情见 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md#打散读热点)。
 
+### `tidb_load_binding_timeout` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 默认值：`"200"`
+- 范围：`(0, 2147483647]`
+- 单位：秒
```
Comment (hfxsd) [inline system-variables.md:2920]: 英文版写的是毫秒

File: system-variables.md [hunk 9]
```diff
@@ -2910,6 +2910,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 这个变量用来设置基于负载的 replica read 的触发阈值。当 leader 节点的预估排队时间超过阈值时，TiDB 会优先从 follower 节点读取数据。格式为时间，例如 `"100ms"` 或 `"1s"`。详情见 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md#打散读热点)。
 
+### `tidb_load_binding_timeout` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 默认值：`200`
+- 范围：`(0, 2147483647]`
+- 单位：秒
```
Comment (hfxsd) [inline system-variables.md:2920]:
```suggestion
- 单位：毫秒
```

File: system-variables.md [hunk 10]
```diff
@@ -2910,6 +2910,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 类型：字符串
 - 这个变量用来设置基于负载的 replica read 的触发阈值。当 leader 节点的预估排队时间超过阈值时，TiDB 会优先从 follower 节点读取数据。格式为时间，例如 `"100ms"` 或 `"1s"`。详情见 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md#打散读热点)。
 
+### `tidb_load_binding_timeout` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 默认值：`200`
```
Comment (hfxsd) [inline system-variables.md:2918]:
```suggestion
- 类型：整数型
- 默认值：`200`
```

**Review by hfxsd — APPROVED** [review]

**Review by songrijie — APPROVED** [review]

File: sql-plan-management.md [hunk 11]
```diff
@@ -476,6 +476,52 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## 利用 Statement Summary 表获取需要绑定的查询
+
+[Statement Summary](/statement-summary-tables.md) 的表中存放了近期的 SQL 相关的执行信息，如延迟、执行次数、对应计划等。你可以通过查询 Statement Summary 表得到符合条件查询的 `plan_digest`，然后[根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定)。
+
+以下示例查找过去 2 周执行次数超过 10 次、执行计划不稳定且未被绑定的 SELECT 语句，按照执行次数排序，将执行次数前 100 的查询绑定到对应的查询延迟最低的计划上。
```
Comment (Oreoxmt) [inline sql-plan-management.md:483]:
```suggestion
以下示例查找过去两周执行次数超过 10 次、执行计划不稳定且未被绑定的 `SELECT` 语句，并按照执行次数排序，将执行次数前 100 的查询绑定到对应的查询延迟最低的计划上。
```

**Review by Oreoxmt — APPROVED** [review]

File: sql-plan-management.md [hunk 12]
```diff
@@ -227,7 +227,7 @@ CREATE BINDING FOR SELECT * FROM t WHERE a > 1 USING SELECT * FROM t use index(i
 以下为根据历史执行计划创建绑定的注意事项：
 
 - 该功能是根据历史的执行计划生成 hint 而实现的绑定，历史的执行计划来源是 [Statement Summary Tables](/statement-summary-tables.md)，因此在使用此功能之前需开启系统变量 [`tidb_enable_stmt_summary`](/system-variables.md#tidb_enable_stmt_summary-从-v304-版本开始引入)。
-- 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，目前还不支持通过历史执行计划进行绑定。
+- 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，自动生成的 hint 不够完备，可能导致无法完全固定住计划，对于这类情况在创建时会产生 warning。
```
Comment (hfxsd) [inline sql-plan-management.md:230]:
```suggestion
- 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，自动生成的 hint 不够完备，可能导致无法完全固定住计划，对于这类情况在创建时会产生告警。
```

File: system-variables.md [hunk 13]
```diff
@@ -2910,6 +2910,17 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+### `tidb_load_binding_timeout` <span class="version-mark">从 v8.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`200`
+- 范围：`(0, 2147483647]`
+- 单位：毫秒
+- 这个变量用来控制 load binding 的超时时间。当 load binding 的执行时间超过该值时，会停止加载。
```
Comment (hfxsd) [inline system-variables.md:2922]:
```suggestion
- 这个变量用来控制加载 binding 的超时时间。当加载 binding 的执行时间超过该值时，会停止加载。
```

**Pull request merged**